.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -361,7 +361,9 @@ jobs:
     #
     # Note we still need rustfmt for the cxx-qt-gen tests
     - name: "Install Rust toolchain"
-      run: rustup component add clippy rustfmt
+      run: |
+        rustup default 1.77.2
```

Comment:
you'll need to do this for other tasks like coverage / build_wasm to be consistent, but lets see what CI thinks of this so far